packages/cmf/src/onError.md

```diff
@@ -65,15 +65,15 @@ in your own components in some key places.
 import { ErrorBoundary } from '@talend/react-cmf';
 
 function DoComplexComponent(props) {
-    // ... do some stuff
+	// ... do some stuff
```

Comment:
4 spaces vs 8 spaces ?

Reply:
not it was tab and now it space, don t know why ... may be a prettier which do not understand this is javascript :trollface: